src/components/AmplitudeAnalytics/TraceEvent.tsx

```diff
@@ -8,6 +8,7 @@ type TraceEventProps = {
   events: Event[]
   name: EventName
   properties?: Record<string, unknown>
+  propertiesList?: Record<string, unknown>[]
```

Comment:
after reading through this i'm not sure i agree a propertiesList should exist. are there other cases where you would want to send many events from one interaction?

Reply:
yeah I think you're right, this was the main use case I can think of, but I agree w you that sending one event w a token list is better than multiple events, so we can see the aggregation.